lxd/instance/drivers/driver_lxc.go

```diff
@@ -8330,6 +8330,12 @@ func (d *lxc) Metrics(hostInterfaces []net.Interface) (*metrics.MetricSet, error
 		d.logger.Warn("Failed to get oom kills", logger.Ctx{"err": err})
 	}
 
+	// If we failed to get OOM kills, because of a couple of reasons (instance stopped, cgroup controller not available, etc),
+	// we default to 0 instead of -1 for the MemoryOOMKillsTotal metric (a total of `-1` would be misleading).
+	if oomKills == -1 {
```

Comment:
Lets use defensive programming technique and use `if oomKills < 0 `

Reply:
fixed